tracer/src/Datadog.Trace/Debugger/Upload/SymbolUploadApi.cs

```diff
@@ -38,7 +38,7 @@ private SymbolUploadApi(
         {
             _apiRequestFactory = apiRequestFactory;
             _eventMetadata = eventMetadata;
-            _enableCompression = enableCompression;
+            _enableCompression = false;
```

Comment:
nit: It seems strange to force this to `false` _here_ 😕 why not force it in the settings? At least a comment explaining why would probably be useful 😅

Reply:
I did it also in the settings but I'm too paranoid now 😅
I can revert it if it's too much.